tools.code-workspace

```diff
@@ -3,6 +3,9 @@
 		{
 			"path": "packages/analyzer"
 		},
+	        {
```

Comment:
Can you sort the whole list while you're here?

Reply:
Done. I don't actually even know what this file does.